examples/set-label/.expected/diff.patch

```diff
@@ -0,0 +1,55 @@
+diff --git a/configs/example-config.yaml b/configs/example-config.yaml
+index 1c43349..02e8ab4 100644
+--- a/configs/example-config.yaml
++++ b/configs/example-config.yaml
+@@ -2,7 +2,11 @@ apiVersion: v1
+ kind: ConfigMap
+ metadata:
+   name: the-map
++  labels:
++    color: orange
++    fruit: apple
+ data:
++  fruit: apple
```

Comment:
what's this being changed?

Reply:
Because there is a custom field spec for this.

Reply:
Not following, what is a function for setting labels changing fields that are not labels?

Reply:
This is used to test the custom field specs. Users can use it to add labels to any field in any resource.

Reply:
Even though this is technically true, I think saying that ConfigMap.data is a label creates confusion. Instead use a custom resource with a label field.

Reply:
Updated the test case.